src/main/java/io/jenkins/plugins/gitlabserver/servers/GitLabServer.java

```diff
@@ -60,16 +66,17 @@
             "scm.",
             "source."
     };
+
     /**
-     * Optional name to use to describe the end-point.
+     * A unique name to use to identify the end-point.
```

Comment:
Can you change this to `A unique name used to identify the endpoint.`?

Reply:
Done.